config/ModulesMapping.jsonc

```diff
@@ -19,7 +19,7 @@
   "Files": "^drives\\.|^shares\\.|^users.drive$|^groups.drive$",
   "Financials": "^financials\\.",
   "Groups": "^groups.group$|^groups.directoryObject$|^groups.conversation$|^groups.endpoint$|^groups.extension$|^groups.groupLifecyclePolicy$|^groups.resourceSpecificPermissionGrant$|^groups.profilePhoto$|^groups.conversationThread$|^groupLifecyclePolicies\\.|^users.group$|^groups.directorySetting$|^groups.Actions$|^groups.Functions$|^groupSettings\\.|^groups.groupSetting$|^groupSettingTemplates\\.",
-  "Identity.DirectoryManagement": "^administrativeUnits\\.|^contacts\\.|^devices\\.|^domains\\.|^directoryRoles\\.|^directoryRoleTemplates\\.|^directorySettingTemplates\\.|^settings\\.|^subscribedSkus\\.|^contracts\\.|^directory\\.|^users.scopedRoleMembership$|^organization.organization$|^organization.organizationalBranding$|^organization.organizationSettings$|^organization.Actions$|^organization.extension$|^tenantRelationships.Actions$|admin.peopleAdminSettings$",
+  "Identity.DirectoryManagement": "^administrativeUnits\\.|^contacts\\.|^devices\\.|^domains\\.|^directoryRoles\\.|^directoryRoleTemplates\\.|^directorySettingTemplates\\.|^settings\\.|^subscribedSkus\\.|^contracts\\.|^directory\\.|^users.scopedRoleMembership$|^organization.organization$|^organization.organizationalBranding$|^organization.organizationSettings$|^organization.Actions$|^organization.extension$|^tenantRelationships.Actions$|admin.peopleAdminSettings$|^organization.partnerInformation$",
```

Comment:
You need to escape the `.` character since it has a special meaning in a regular expression. Leaving it unescaped will match any single character.

Reply:
Anchors have been provided to match the exact tag with what is available in the overall open Api file.
<img width="599" alt="image" src="https://github.com/microsoftgraph/msgraph-sdk-powershell/assets/10947120/1cb66300-3270-4639-9637-afe4931fafaa">
If I escape the dot character as you are suggesting, there is a possibility of just adding any paths containing that tag name yet I want an exact path. 
See the "CreatePredicate" function [here](https://github.com/microsoftgraph/microsoft-graph-devx-api/blob/dev/OpenAPIService/OpenApiService.cs) from line 166.

Reply:
I don't understand what you mean exactly.
`^organization.partnerInformation$` will match `organization.partnerInformation`, I agree, but it will also match `organization_partnerInformation`, `organization1partnerInformation`, `organizationxpartnerInformation` and anything else in place of the `.` which I don't think is the intent here unless I'm mistaken

You can test it out using the javascript engine or any other engine you have available to you.

What you want is probably `^organization\\.partnerInformation$`